output.tf

```diff
@@ -0,0 +1,7 @@
+output "name" {
+  value = "${azurerm_template_deployment.subscription.name}"
+}
+
+output "path" {
+  value = "${azurerm_template_deployment.subscription.outputs["resourceId"]}"
```

Comment:
Why return this?

Reply:
So that it could be used in connection string.

Reply:
how?

Reply:
For example:
```
    SERVICE_BUS_CONNECTION_STRING = "${module.servicebus-topic.primary_send_and_listen_connection_string}/${module.servicebus-subscription.path}"
```

Reply:
That connection string will be invalid. We only need subscription name to create the connection string, which we're doing already.